src/App.js

```diff
@@ -17,16 +27,52 @@ function App() {
     previous: "50 kg x 5",
     kg: "50",
     reps: "9",
-    hola: "bye",
   };
 
+  const listOfButtons = {
+    set: "ADD SET",
+    exercise: "ADD EXERCISE",
+    cancel: "CANCEL WORKOUT"
```

Comment:
This logic is probably better off inside of the Button component. It makes the main App component bigger, and there's no real good reason to send it down as props because the information does not change and is not dynamic.